src/libraries/System.Reflection.Metadata/tests/Metadata/Decoding/CustomAttributeDecoderTests.cs

```diff
@@ -11,7 +12,7 @@ namespace System.Reflection.Metadata.Decoding.Tests
 {
     public class CustomAttributeDecoderTests
     {
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.HasAssemblyFiles))]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsMonoRuntime))]
```

Comment:
This is going to break the extra-platforms run where we test things that return an empty string for Assembly.Location, it's not Mono, and this conditional is guarding that.

Do this instead?

```suggestion
        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.HasAssemblyFiles))]
        [ActiveIssue("https://github.com/dotnet/runtime/issues/73593", TestRuntimes.Mono)]
```

Reply:
I am actually trying to run it only on Mono because reflection based test that checking all attributes is failing on Mono, so leaving this old hard coded test only for Mono. 


Though I was mistakenly removed the `nameof(PlatformDetection.HasAssemblyFiles) ` from platforms list, will add back